eclass/coreos-kernel.eclass

```diff
@@ -128,6 +128,13 @@ coreos-kernel_src_prepare() {
 coreos-kernel_src_configure() {
 	# Use default for any options not explitly set in defconfig
 	yes "" | kmake oldconfig
+
+	# For convinence, generate a minimal defconfig of the build
+	local f=$(find_defconfig)
+	f=${f##/*/}
+	kmake savedefconfig
+	einfo "Saving minimal kernel defconfig as ${KBUILD_OUTPUT}/$f"
```

Comment:
I'd just leave it as defconfig instead of matching the config in the ebuild tree, a number of names are searched so it isn't as predictable as "{KBUILD_OUTPUT}/defconfig"